console.rst

```diff
@@ -641,6 +641,14 @@ profile is accessible through the web page of the profiler.
     terminal supports links). If you run it in debug verbosity (``-vvv``) you'll
     also see the time and memory consumed by the command.
 
+.. caution::
+
+    If you use Messenger component and want to profile the ``messenger:consume``
+    command please be aware that it will only create the profile only when
```

Comment:
2 "only" in same sentence seem unatural to me (I am non native English speaker)

what about:

please be aware that the profile will only be done with the --no-reset option

Reply:
We did this reword while merging. Thanks.